File: HelperJobby/DTOs/User/UserDTO.cs
using HelperJobby.DTOs.Account;

namespace HelperJobby.DTOs.User;

public class UserDTO
{
    public int Id { get; set; }

    public string Email { get; set; }

    public string Password { get; set; }

    public string AccountType { get; set; }

    public JobSeekerDTO JobSeekerAccount { get; set; }

    public EmployerDTO? EmployerAccount { get; set; }
}

File: HelperJobby/client-app/src/DTOs/userRelatedDTOs/UserDTO.ts
import {JobSeekerDTO} from "../accountDTOs/JobSeekerDTO";
import {EmployerDTO} from "../accountDTOs/EmployerDTO";

export interface UserDTO {
    id: number;
    email: string;
    password: string;
    accountType: string;
    jobSeekerAccount: JobSeekerDTO;
    employerAccount: EmployerDTO;
}